packages/ui/src/overview/hooks/useDeadlines.ts

```diff
@@ -1,26 +1,34 @@
 import { useMemo } from 'react'
 
 import { useLocalStorage } from '@/common/hooks/useLocalStorage'
+import { useElectionStage } from '@/council/hooks/useElectionStage'
 import { asElection } from '@/council/types/Election'
+import { Member } from '@/memberships/types'
 import { useGetAllDeadLinesQuery } from '@/overview/queries'
 import { asUpcomingWorkingGroupOpening } from '@/working-groups/types'
 
 interface UseDeadlinesParams {
   groupId?: string
+  member: Member
 }
 
-export type DeadlineNamespace = 'election' | 'proposals' | 'openings' | 'upcomingOpenings'
+export type DeadlineNamespace = 'elections' | 'proposals' | 'openings' | 'upcomingOpenings'
 
 const initializer: Record<DeadlineNamespace, string[]> = {
   proposals: [],
   openings: [],
   upcomingOpenings: [],
-  election: [],
+  elections: [],
 }
 
-export const useDeadlines = (params?: UseDeadlinesParams) => {
-  const options = params?.groupId || {}
-  const { loading, data } = useGetAllDeadLinesQuery(options)
+export const useDeadlines = (params: UseDeadlinesParams) => {
+  const variables = {
+    groupId: params.groupId || {},
+    proposalCreator: params.member.isCouncilMember ? undefined : { id_eq: params.member.id },
+    isLead: false,
```

Comment:
I meant this `isLead: false` as a placeholder but we actually need to add some logic to determine whether the member is the lead of some WG and if so which ones, and finally pass the ids. e.g:
```suggestion
    isLead: groupIds.length > 0,
    group: groupIds.length > 0 ? { id_in: groupIds } : undefined,
```